script/s1perf.sh

```diff
@@ -0,0 +1,7 @@
+rm -rf ~/ccfs3tmp
+mkdir -p $HOME/logs
+LOGFILE="$HOME/logs/log$(date -j +'%Y%m%d_%H%M%S').txt"
+echo Logging to $LOGFILE
+# STATIONS="SBC,RIO,DEV,HEC,RLR,SVD,RPV,BAK,CBC,CCA,CCC,CDD,HLL,HOL,AGM,AGO"
+STATIONS="SBC,RIO,DEV,HEC,RLR,SVD,RPV,BAK"
+noisepy cross_correlate --raw_data_path ~/s3tmp/scedc/2022/002 --ccf_path ~/ccfs3tmp --freq_norm rma --stations "$STATIONS" --xml_path ~/s3tmp/FDSNstationXML/ 2>&1 | tee $LOGFILE
```

Comment:
I do not see any problem with te lines in S1_fft_cc_MPI. I wonder if the result of this test gives data that looks like cross correlations. When we use the plotting commands in the get_started tutorial:
file = os.path.join(cc_data_path, '2019_02_01_00_00_00T2019_02_01_12_00_00.h5')
plotting_modules.plot_substack_cc(file,0.1,1,200,False)
does it plot what it did before?

Reply:
I ran the get_started tutorial and as far as I can tell the plots look the same as before. Couple of examples:

<img width="975" alt="image" src="https://github.com/mdenolle/NoisePy/assets/10456546/a2bf245d-47ca-4fc3-8f6f-b0d6537d0d41">
<img width="982" alt="image" src="https://github.com/mdenolle/NoisePy/assets/10456546/357c80f0-34a0-4c7e-9201-f3ca45c094f5">

Can you confirm?